agents/property.json

```diff
@@ -680,6 +680,17 @@
                                     }
                                 ]
                             }
+                        ],
+                        "cmd": [
+                            {
+                                "name": "on_user_joined",
```

Comment:
I think it will be better to have ref count when user join or leave.

Reply:
added